src/pages/_app.tsx

```diff
@@ -1,6 +1,18 @@
 import { AppProps } from 'next/app';
+import { useState } from 'react';
+import { SplashScreen } from '../components/SplashScreen';
 
 const ClassifyApp = ({ Component, pageProps }: AppProps): JSX.Element => {
+  const [isSplashShown, setIsSplashShown] = useState(false);
+
+  const handleSplash = () => {
+    setIsSplashShown(true);
+  };
+
+  if (!isSplashShown) {
+    return <SplashScreen handleSplash={handleSplash} />;
+  }
+
   return <Component {...pageProps} />;
```

Comment:
```suggestion
  return (
    <>
      {!isSplashShown && <SplashScreen onSplashEnd={handleSplashEnd} />}
      <Component {...pageProps} />
    </>;
```

이렇게 하면 페이지의 로딩을 같이 할 수 있어서 스플래시 이후 빠르게 화면을 보여줄 수 있겠습니다!

Reply:
넵! 로딩도 잡고 버벅거림도 없네여.. 100점.. 반영했습니다